src/pages/Product/components/ColorButton/ColorButton.js

```diff
@@ -0,0 +1,15 @@
+import React, {useState} from "react";
+import "../Color/Color.scss";
+
+const ColorButton = (props) => {
+  const {color} = props
```

Comment:
```suggestion
  const {color, setColor} = props
```
props로 전달해 주는 데이터가 두가지이기에 둘다 구조분해 할당을 해주면 좋겠네요!

Reply:
수정했습니다~ 확인 부탁드립니다
감사합니다 💕